Dockerfile

```diff
@@ -21,7 +19,7 @@ RUN	CGO_ENABLED=0 GOOS=linux go build -mod vendor -ldflags \
     "-w -s -X sigs.k8s.io/aws-encryption-provider/pkg/version.Version=$TAG" \
     -o bin/aws-encryption-provider cmd/server/main.go
 
-FROM busybox AS aws-encryption-provider
-COPY --from=build /etc/ssl/certs/ /etc/ssl/certs/
+FROM public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base:latest
 COPY --from=build /go/src/sigs.k8s.io/aws-encryption-provider/bin/aws-encryption-provider /aws-encryption-provider
-ENTRYPOINT ["/aws-encryption-provider"]
+COPY --from=go-runner /go-runner /go-runner
+ENTRYPOINT ["/aws-encryption-provider"]
```

Comment:
Why are you baking in go-runner if you are not using it for your entrypoint? Also, the go-runner image is basically just minimal + go-runner binary. You should be able to just use go-runner as a base if you need it

Reply:
there are still some tzdata, certs, etc on the minimal base. We feel it would make it simpler to use the public ecr based minimal image for consistency across projects and ability to change if needed at all.

Reply:
it also gives us the flexibility to use non root minimal image further. We haven't done/tested non root image as part of this PR.

Reply:
The go-runner is built on top of base (including tzdata, cacerts), so my point is you should just use go-runner to get the base, rather than extracting the binary out of go runner

Reply:
sounds good, we can do that. How about the argument for using non root. Does go-runner have a non root image?

Reply:
Plus, all you have to do to make an image nonroot is add a user at the end
```
USER 65532
```

Reply:
ok. thats simple then..we will change base to go-runner